src/main/webapp/app/exercises/shared/dashboards/tutor/exercise-assessment-dashboard.component.html

```diff
@@ -563,6 +565,7 @@ <h4 class="d-inline-block">
                                             [routerLink]="getAssessmentLink(submissionWithComplaint.submission)"
                                             [queryParams]="getComplaintQueryParams(submissionWithComplaint.complaint)"
                                             class="btn btn-success btn-sm"
+                                            id="evaluate-complaint"
```

Comment:
Maybe we could add prefixes for ids if they are used for cypress tests so that it is clear that they are used for cypress. 
e.g. `cy-evaluate-complaint`